sysutils/entr/Portfile

```diff
@@ -19,23 +21,11 @@ long_description    a utility for running arbitrary commands when \
 homepage            http://entrproject.org
```

Comment:
the `homepage` redirects to https://eradman.com/entrproject/, probably good to update this (and of course double-check the `master_sites` to avoid this

Reply:
or just move to GitHub...

Reply:
Thanks for reviewing @reneeotten.  Changed it to use that URL for the homepage, and pulling source using the `github` portgroup.